pkg/route/engine.go

```diff
@@ -634,6 +634,20 @@ func (engine *Engine) recv(ctx *app.RequestContext) {
 	}
 }
 
+// HandleContext Re-enter a context that has been rewritten.
+// This can be done by setting ctx.Request.SetRequestURI() to new target and response will be reset.
```

Comment:
The comment can add an example of the use, otherwise it is very easy to dead loop

Reply:
done